.goreleaser.yaml

```diff
@@ -0,0 +1,87 @@
+project_name: apm-lambda-extension
+
+before:
+  hooks:
+    - make check-licenses
+    - make NOTICE.txt
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w
+    binary: "extensions/{{ .ProjectName }}"
+
+archives:
+  - id: zip
+    format: zip
+    name_template: |-
+      {{ .Tag }}-{{ .Os }}-{{ .Arch }}
+    files:
```

Comment:
**issue**: I think you also need to apply mtime to files. `builds_info` only applies to binaries afaics.

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/645fcfddc05f72acb45db59c1e6f20b4aa18924f